app/models/validations/sales/household_validations.rb

```diff
@@ -46,6 +47,21 @@ def validate_person_age_and_relationship_matches_economic_status(record, person_
     end
   end
 
+  def validate_person_age_matches_economic_status(record, person_num)
+    age = record.public_send("age#{person_num}")
+    economic_status = record.public_send("ecstat#{person_num}")
+    return unless age && economic_status
+
+    if age < 16 && !tenant_is_economic_child?(economic_status)
+      record.errors.add "ecstat#{person_num}", I18n.t("validations.household.ecstat.child_under_16", person_num:)
+      record.errors.add "age#{person_num}", I18n.t("validations.household.age.child_under_16", person_num:)
+    end
+    if tenant_is_economic_child?(economic_status) && age > 16
+      record.errors.add "ecstat#{person_num}", I18n.t("validations.household.ecstat.child_over_16", person_num:)
+      record.errors.add "age#{person_num}", I18n.t("validations.household.age.child_over_16", person_num:)
+    end
+  end
```

Comment:
given this is adding the same errors as far as I can see, is there a reason you have chosen to have two separate ifs rather than `if (under 16 and !child) || (child and over 16)`
Also as I write this I realise the conditions are for over 16 and under 16, is it the spec that 16 years old can be either?

Reply:
Hmm, so the first validation is if the tenant is under 16 but not marked as a child, the second one is it they are over 16 and marked as a child. I guess it's different situations/different error messages etc

Reply:
my mistake. I thought I had read the yml directions carefully and they were the same